ethereum-rpc/src/constructors/express/routes/api/v1/uniswap.js

```diff
@@ -26,58 +47,34 @@ module.exports = ({
           return res.status(400).json({ errors: errors.array() });
         }
         const { address = "*" } = req.params;
-        const { privateKey = "", amount = "", tokenFrom = "", tokenTo = "" } = { ...req.query };
-        const tokenType = tokenTo.toString().toUpperCase();
+        const { privateKey = "", amount = "", tokenFrom = "", tokenTo = "", tokenToAddress = "", network = "" } = { ...req.query };
+        if (tokenFrom === tokenTo) {
+          res.status(500).json({ error: "tokenFrom and tokenTo cannot be same" })
+        }
```

Comment:
This check can be done in a custom express-validator function

Reply:
done.